pkgs/applications/graphics/yed/default.nix

```diff
@@ -11,7 +11,7 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ makeWrapper unzip wrapGAppsHook ];
   # For wrapGAppsHook setup hook
-  buildInputs = [ jre.gtk3 ];
+  buildInputs = [ jre.gtk3 or gtk3 ];
```

Comment:
Wouldn't it be a better idea to use gtk3 on darwin and jre.gtk3 on all other platforms explicitly?